index.html

```diff
@@ -44,5 +44,160 @@ <h2 class="connect-txt">LET'S CONNECT</h2>
             </div>
         </div>   
     </header> 
+    <!-- projects section -->
+    <section class="projects-container">
+        <div class="project">
+            <img src="./images/project1.png" class="project-img" alt="">
+            <div class="project-txt">
+                <h1 class="project-title">Tonic</h1>
+                <div class="caption">
+                    <h3 class="canopy">CANOPY</h3>
+                    <img src="./images/Counter.png" alt="">
+                    <h3>Back end dev</h3>
+                    <img src="./images/Counter.png" alt="">
+                    <h3>2015</h3>
+                </div>
+                <p class="text">
+                    A daily selection of privately personalized reads;
+                    no accounts sign-ups required.
+                </p>
+                <ul class="languages">
+                    <li>Html</li>
+                    <li>Css</li>
+                    <li>JavaScript</li>
+                </ul>
+                <button>See Project</button>
+            </div>
+        </div>
+        <div class="project">
+            <img src="./images/project2.png" class="project-img" alt="">
+            <div class="project-txt">
+                <h1 class="project-title">Multi-Post Stories</h1>
+                <div class="caption">
+                    <h3 class="canopy">CANOPY</h3>
+                    <img src="./images/Counter.png" alt="">
+                    <h3>Back end dev</h3>
+                    <img src="./images/Counter.png" alt="">
+                    <h3>2015</h3>
+                </div>
+                <p class="text">
+                    A daily selection of privately personalized reads;
+                    no accounts sign-ups required.
+                </p>
+                <ul class="languages">
+                    <li>Html</li>
+                    <li>Css</li>
+                    <li>JavaScript</li>
+                </ul>
+                <button>See Project</button>
+            </div>
+        </div>
+        <div class="project">
+            <img src="./images/project3.png" class="project-img" alt="">
```

Comment:
- [ ] Kindly add meaningful `alt` to your images so that your page becomes more accessible by screen readers.